source/core/src/main/com/csse3200/game/entities/factories/PlayerFactory.java

```diff
@@ -30,64 +29,65 @@
  * the properties stores in 'PlayerConfig'.
  */
 public class PlayerFactory {
-    private static final PlayerConfig stats =
-            FileLoader.readClass(PlayerConfig.class, "configs/player.json");
+    private static final PlayerConfig stats = FileLoader.readClass(PlayerConfig.class, "configs/player.json");
+
+    public static Entity createPlayer(GdxGame game) {
+        String imagePath = AnimalSelectionActions.getSelectedAnimalImagePath();
+        InputComponent inputComponent = ServiceLocator.getInputService().getInputFactory().createForPlayer();
 
-  /**
-   * Create a player entity.
-   * @return entity
-   */
-  public static Entity createPlayer(GdxGame game) {
-    String imagePath = AnimalSelectionActions.getSelectedAnimalImagePath();
-    InputComponent inputComponent =
-        ServiceLocator.getInputService().getInputFactory().createForPlayer();
+        Entity player = new Entity()
+//                .addComponent(new TerrainLoaderComponent())
```

Comment:
Should this be commented out or removed entirely? I'd imagine that it shouldn't affect this pr and should be whatever is in main.